.gitignore

```diff
@@ -185,8 +185,8 @@ copier_scripts/
 /cms/jinja2/layout
 /cms/static_compiled
 
-node_modules/
-static/
+/node_modules/
```

Comment:
Any reason not to leave the `node_modules` greedy? That is, would we ever want to commit `node_modules` from anywhere in the tree?

Reply:
Agree we would never want to commit `node_modules`.

Reply:
This is less of an immediate problem, admittedly, but related. Ultimately, I think it's reasonable for an app to have a `node_modules` directory where it needs generate complex JS for Wagtail or something (something that might not be bundled up with the global FE media).

Reply:
In that scenario we can add an extra endpoint in webpack that targets a different js file, e.g. `admin.js`.

Reply:
@ababic while the scenario you outline is reasonable, I'd rather we moved this further up (ie. the project `package.json`, `static_src` etc). Or even better, publish it to PyPI if allowed

Reply:
Okay, happy to revert this one.

Reply:
Reverted in b1f0308ff6b14277531ec440e2c86d2e241d106e